kyuubi-hive-jdbc-shaded/pom.xml

```diff
@@ -74,10 +74,15 @@
                                 <exclude>LICENSE.txt</exclude>
                                 <exclude>NOTICE.txt</exclude>
                                 <exclude>mozilla/**</exclude>
+                                <exclude>arrow-git.properties</exclude>
```

Comment:
`META-INF/native-image/**` should be excluded as well, because the shade plugin can not handle it properly.

Reply:
updated.